mdsuite/database/mdsuite_properties.py

```diff
@@ -64,8 +64,8 @@ class MDSuiteProperties:
     kinaci_heat_current = PropertyInfo("Kinaci_Heat_Current", 3)
     charge = PropertyInfo("Charge", 1)
     energy = PropertyInfo("Energy", 1)
-    kinetic_energy = PropertyInfo("KE", 1)  # TODO rename, also in calculators
-    potential_energy = PropertyInfo("PE", 1)  # TODO rename, also in calculators
+    kinetic_energy = PropertyInfo("Kinetic_Energy", 1)
+    potential_energy = PropertyInfo("Potential_Energy", 1)
```

Comment:
this has been changed in the calculators prior to this to use the dataclass?

Reply:
the calculators already use the dataclass member. The strings only show up here

Reply:
perfect! Very nice change!